### ISSUE TITLE
Separate display of same coins

### ISSUE THREAD

**Issue by kiwi810** [foxfolio/foxfolio-desktop#10]
This is an amazing app, thank you. I just hope you keep maintaining it and not abandon like most of the others out there.

It would be nice to see each purchase (or manual purchase entry) of a coin such as BTC at different dates with the different profit margins from the date of purchase at the purchase price entered, instead of combining all BTC in one and showing total value.

Im just looking on keep tabs on different buys and different dates with the appreciate/depreciation of value.
I really hope this isnt difficult to do.
This should all be manual entries

**Comment by greimela** [comment]
First of all: Thanks for the kind words, good to know that someone actually uses this 😄 

I looked into implementing an automated trade history in the past, as it is also suggested in #5.
However, it was considerably more effort than I expected, especially since one had to get past coin prices (relative to BTC/fiat) for each purchase.
That's why I didn't pursue it further.

Would a first version based on manual entry of purchases and prices be enough for you?
That should be easier to build and it could be later extended using automatic data directly from the exchanges.

**Comment by kiwi810** [comment]
Yes that would be great, however what i meant about auto-calculating profits is:

1. Based on a "manually entered previous price" should calculate the current profit/loss (meaning it would compare to current price) - i am not sure where you get the average price at the moment, is it CMC?
2. All prices that it gets and compares to should be taken from for example, CMC / Coingecko / Livecoinwatch, etc
3. I personally am not interested in a "per exchange" basis to capture pricing etc.  As a user like me could use MANY different exchanges and struggle to keep up, so the best would be to capture each coin purchased at a specific price, the date(for interest sake) and then shows the profit/loss, Also the Name of the exchange so the user does not lose track.

I hope that would be easy for you to do.

Another 2 things ive noticed.
1. It would be great if we could export all transactions done to a csv file. SO a backup/restore feature.
2. Dogecoin does not show any data when i added it as far as i have checked.
